### ISSUE TITLE
Error during packaging CFN template resource if there is another file in folder

### ISSUE THREAD

**Issue by JIoJIaJIu** [aws-amplify/amplify-cli#1586]
**Describe the bug**
I'm using vim so it creates temporary files into folder with edited file that leads to error on pushing cfn templates to `S3`

**To Reproduce**
1. Keep editing any file in `vim` during pushing to `S3`
2. 
```
| Function | EduWayLMSEmployeeResolver | Update    | awscloudformation |
? Are you sure you want to continue? Yes
[ '.EduWayLMSEmployeeResolver-cloudformation-template.json.swp', 'EduWayLMSEmployeeResolver-cloudformation-template.json' ]
Only one CloudFormation template is allowed in the resource directory
/home/jiojiajiu/workspace/NewMind/EduWay/sources/VirtualAngelHack2019/amplify/backend/function/EduWayLMSEmployeeResolver
✖ An error occurred when pushing the resources to the cloud
```

**Expected behavior**
Be able to upload template successfully
[Here](https://github.com/aws-amplify/amplify-cli/blob/e022362a44cf698dfd627ab38a5ba9ca9935be18/packages/amplify-provider-awscloudformation/lib/push-resources.js#L234) can apply regexp as `/\.(json|yml|yaml)$/`
If it's ok I can provide PR

**Comment by kaustavghosh06** [comment]
@JIoJIaJIu Would be greaat if you could provide a PR for it.

**Comment by kaustavghosh06** [comment]
Fix has been merged and published in the latest version of the CLI - 1.7.1.
